src/server.js

```diff
@@ -91,6 +92,20 @@ function init(socket, client, token) {
 		socket.emit("auth");
 		socket.on("auth", auth);
 	} else {
+		var ip = socket.handshake.headers['x-forwarded-for'] || socket.request.connection.remoteAddress;
```

Comment:
I get the impression, reading [shout documentation](http://shout-irc.com/docs/) and the way it is designed that deployment without reverse-proxy is one of the options offered, in this case, `x-forwarded-for` can be spoofed by browser ; dare to add in this PR, an option to settings to declare that shout is being ran behind a reverse proxy ?